kymatio/scattering2d/scattering2d.py

```diff
@@ -92,6 +92,8 @@ def __init__(self, J, shape, L=8, max_order=2, pre_pad=False):
         self.pre_pad = pre_pad
         self.max_order = max_order
         self.shape = shape
+        if 2**J>shape[0] or 2**J>shape[1]:
+            raise (RuntimeError('The largest dimension should be smaller than 2^J'))
```

Comment:
should be greater than

Reply:
you mean like "2**J>=shape[0] or 2**J>=shape[1]" ?! Isn't it precisely the setting you're trying to use(and that was available with prior scatnet software)?

Reply:
my bad